utoipa-gen/src/component/schema.rs

```diff
@@ -1595,7 +1595,18 @@ impl ToTokens for SchemaProperty<'_> {
                         }
                     }
                     // TODO support for tuple types
-                    ValueType::Tuple => (),
+                    ValueType::Tuple => {
+                        // Detect unit type ()
+                        if type_tree.children == None {
+                            tokens.extend(quote!{
+                                utoipa::openapi::ObjectBuilder::new()
+                                    .nullable(true)
+                                    .default(Some(serde_json::Value::Null))
+                            })
+                        }
+
+                        ()
```

Comment:
I believe it is enough to add semicolon here at the end of the if.
```suggestion
                        };
```